src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3040,6 +3040,37 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <ReferenceComWrappersToCopyLocal Include="@(ComReferenceWrappers)" Condition="'%(ComReferenceWrappers.CopyLocal)'!='false'"/>
     </ItemGroup>
 
+  </Target>
+
+    <!--
+    ============================================================
+                                        GetAssembliesMetadata
+
+    Resolve Assembly attributes for COM assemblies
+
+        [IN]
+        @(ReferencePath) - Paths to COM referenced wrappers.
+
+        [OUT]
+        @(AssembliesMetadata) - COM assemblies attributes.
+    ============================================================
+    -->
+
+  <Target
+      Name="GetAssembliesMetadata"
+      Condition="'@(ReferencePath)'!=''"
+      Returns="@(AssembliesMetadata)"
+      DependsOnTargets="ResolveComReferences"
+        >
+
+    <GetAssembliesMetadata
+          AssembyPaths="@(ReferencePath)"
+          ContinueOnError="$(ContinueOnError)">
+
+      <Output TaskParameter="AssembliesMetadata" ItemName="_AssembliesMetadata"/>
+
+    </GetAssembliesMetadata>
+
```

Comment:
Some questions:

- Does CSProj not ship its own design-time targets? I assume not, given the question.
- Is .NET Project System guaranteed to be installed everywhere that CSProj is?
- Is CSProj loading the .NET Project System's design time targets already? 
- Would this data be useful for the .NET Project System too? I don't see the context behind this addition.

In general we think of these targets as having a 1:1 correspondence with the binaries we ship. I feel it would be cleaner/safer to put CSProj-specific targets in a CSProj-specific design time targets file, but maybe I'm missing something.

Reply:
Thank you. I'll defer to @YuliiaKovalova for authoritative answers. As far as I can tell CSProj doesn't have its own targets.

(Context: CSProj is currently doing some assembly metadata scanning on the main VS thread. This work is addressing the issue by adding a design-time target+task as a replacement.)

Reply:
To my best knowledge, csproj doesn't have any targets.